src/sinks/clickhouse/native/service.rs

```diff
@@ -1,10 +1,14 @@
 use super::convert::build_block;
-use crate::event::{EventStatus, LogEvent};
+use crate::event::EventStatus;
 use clickhouse_rs::{types::SqlType, Pool};
 use futures::future::BoxFuture;
 use std::task::{Context, Poll};
+use tower::Service;
+use vector_common::finalization::{EventFinalizers, Finalizable};
+use vector_common::request_metadata::{MetaDescriptive, RequestMetadata};
 use vector_common::{internal_event::CountByteSize, Error};
-use vector_core::{stream::DriverResponse, ByteSizeOf};
+use vector_core::event::Event;
+use vector_core::stream::DriverResponse;
 
 pub(super) struct ClickhouseResponse {
```

Comment:
Perhaps we should rename this to `NativeClickhouseResponse`, for consistency ?

Reply:
Good point, fixed.